File: C/621.c
#include <stdio.h>
#include <stdlib.h>

// 621 - La Otra Página - Iván - https://github.com/wildfireOfMine

int main()
{
    int loop;
    scanf("%d", &loop);
    int i;
    int page;
    for (i = 0;i<loop; i++) {
        scanf("%d", &page);
        if (page % 2 == 0) {
            printf("%d\n", page+1);
        } else {
            printf("%d\n", page-1);
        }
    }
    return 0;
}

File: Java/621.java
import java.util.Scanner;

/**
 *
 * 621 - La Otra Página - Iván - https://github.com/wildfireOfMine
 */
public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int prueba = sc.nextInt();
        for (int i = 1; i <= prueba; i++) {
            int pagina = sc.nextInt();
            if (pagina % 2 == 0) {
                pagina++;
            } else {
                pagina--;
            }
            System.out.println(pagina);
        }
    }
}
